### ISSUE TITLE
Dynamic usage of icons don`t accept any props in 3.2.x

### ISSUE THREAD

**Issue by diego-menezes** [tabler/tabler-icons#1090]
In menu on my app i pass a icon on prop and declare dinamicaly like above

```
const acao = {
url: "#",
nome:  "Home"
icone: IconHome
}

    <NavLink
.....
      component={Link}
      href={acao.url}
      label={acao.nome}
      leftSection={<acao.icone size="2rem" stroke={1.5}/>}
....
    />
```

But now in 3.2 i got this error because size, stroke or any other props.

```
TS2322: Type { size: string; stroke: number; } is not assignable to type IntrinsicAttributes & RefAttributes<Icon>
Property size does not exist on type IntrinsicAttributes & RefAttributes<Icon>
```

but work on npm run dev, but don`t on start...

**Comment by Fedec96** [comment]
I think this might be a duplicate of #1077 
Should be fixed when #1078 gets merged

**Comment by BG-Software-BG** [comment]
#1078 is already merged.  
Is this issue resolved with that?
